modules/dynamic-apiserver/pkg/cli/install_operation.go

```diff
@@ -293,7 +293,7 @@ func runInstall(k8sclient kubernetes.Interface) func(command *cobra.Command, arg
 			return err
 		}
 		command.Println("Waiting for Kuberlogic to be ready...")
-		cmd = fmt.Sprintf("%s -n kuberlogic wait --for=condition=ready pods --all --timeout=300s", kubectlBin)
+		cmd = fmt.Sprintf("%s -n kuberlogic wait --for=condition=ready pods --all --timeout=600s", kubectlBin)
```

Comment:
Please comment why timeout increased

Reply:
Sometimes 300 is not enough on this tests:
https://github.com/AvoidMe/kuberlogic/actions/runs/3085186362/jobs/4988330890